x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/pipelines/fetch_ml_inference_pipeline_processors.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// import { InferencePipeline } from '../../../../../common/types/pipelines';
```

Comment:
be sure to remove this line (or update it) and all `TODO` comments as well ;) (I assume you're waiting until the back-end pieces merge for this, but want to make sure :) )

Reply:
yep, once the backend merges I can replace the local types with this one. I was going to add it but deciided to do this to deal with less merge conflicts.

Reply:
I'll get this updated now